build.sbt

```diff
@@ -212,8 +212,7 @@ lazy val pluginTesterScala = Project(id = "akka-grpc-plugin-tester-scala", base
     skip in publish := true,
     crossScalaVersions := Dependencies.Versions.CrossScalaForLib,
     scalaVersion := scala212,
-    ReflectiveCodeGen.codeGeneratorSettings ++= Seq("flat_package", "server_power_apis"),
-    Compile / ReflectiveCodeGen.protocOptions += "--include_std_types")
```

Comment:
No, I get protoc errors saying that `--include_std_types` isn't a flag I can pass. I also saw that the source sbt-protoc documentation mentioned providing them this way: https://github.com/thesamet/sbt-protoc#additional-options 

I could just be wrong or it could be an artifact of the RC.